waterlog/src/Pages/LeakageInfoComponent.js

```diff
@@ -0,0 +1,56 @@
+
+import React, { Component } from 'react';
+import { NavLink } from "react-router-dom";
+
+class LeakageInfoComponent extends Component {
+    section = this.props.section; 
+    severity = this.props.severity; 
+    waterLost = this.props.waterLost;
+    cost = this.props.cost;
+    isMapExpanded = false;
+
+    styles = {
+        mild: {
+            textAlign: 'center',
+            color: 'white'
+        },
+        moderate: {
+            textAlign: 'center',
+            color: 'white'
+        }, 
+        severe: {
+            textAlign: 'center',
+            color:'red'
+        }
+    }
+    expandMap(){
+        console.log('map expanded');
```

Comment:
Remove console logs and use debugger to debug your code: https://code.visualstudio.com/docs/nodejs/reactjs-tutorial#_debugging-react

Reply:
Please also go through the rest of your code and remove console logs, not just this one

Reply:
I removed all the console logs